packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/actions.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Dockerfile in upgrade-test sets:
+# WORKDIR /usr/src/agoric-sdk/
+# Overriding it during development has occasionally been useful.
+SDK=${SDK:-/usr/src/agoric-sdk}
+. $SDK/upgrade-test-scripts/env_setup.sh
+
+# Enable debugging
+set -x
+
+AGUP=$SDK/upgrade-test-scripts/agoric-upgrade-11
```

Comment:
Should this reference 12 now?
```suggestion
AGUP=$SDK/upgrade-test-scripts/agoric-upgrade-12
```